frontend/package.json

```diff
@@ -21,7 +21,7 @@
     "analyze": "yarn build && npx source-map-explorer 'build/static/js/*.js'",
     "build": "export GIT_HASH=$(git rev-parse --short HEAD) || true && cross-env EXTEND_ESLINT=true react-app-rewired build",
     "profile:serve": "yarn build --profile && npx serve -s build",
-    "test": "cross-env REACT_APP_TESTING=true && react-scripts test",
+    "test": "cross-env REACT_APP_TESTING=true TZ=UTC react-scripts test",
```

Comment:
Is the change in global-setup not enough?
Ideally we would want the test to go over mutiple timezones and make sure nothing breaks in any timezone. We need to test for:

- far wes, eg. California Timezone
- europe, eg. Paris timezone
- far east, eg. China timezone
- edge cases, eg. timezone with +Xh30min diff